Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml

```diff
@@ -0,0 +1,3115 @@
+    type: title
+    task:
+      id: 7ef09ff3-e817-4636-8234-feea958b778d
+      version: -1
+      name: Isolate Endpoint
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "87"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 160,
+          "y": 4760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "87":
+    id: "87"
+    taskid: b98a91c6-31c1-4872-8639-80b5d94bc53a
+    type: condition
+    task:
+      id: b98a91c6-31c1-4872-8639-80b5d94bc53a
+      version: -1
+      name: Isolate Endpoint?
+      description: Check if IsolateEndpoint it true.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "136"
+      "Yes":
+      - "102"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.IsolateEndpoint
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 160,
+          "y": 4890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "92":
+    id: "92"
+    taskid: e026a817-ada8-4a83-8ef1-2e26a98fd8c6
+    type: regular
+    task:
+      id: e026a817-ada8-4a83-8ef1-2e26a98fd8c6
+      version: -1
+      name: Set User Containment Executed
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "82"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: RDP_Brute_Force_Investigation.User_Containment
+      value:
+        simple: Executed
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -630,
+          "y": 300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: User Disabled Status
+      output:
+        complex:
+          root: RDP_Brute_Force_Investigation
+          accessor: User_Containment
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "93":
+    id: "93"
+    taskid: c6b2a205-bcd3-4140-8698-3601e1c729ea
+    type: playbook
+    task:
+      id: c6b2a205-bcd3-4140-8698-3601e1c729ea
+      version: -1
+      name: Cortex XDR - Get entity alerts by MITRE tactics
```

Comment:
If we keep it, change the name of the playbook.
Instead of '.... Get entity alerts...' use '...Get username related alerts...'